doc/guides/technical-values.md

```diff
@@ -0,0 +1,66 @@
+# Technical values and priorities
+
+The project uses these technical values to establish priorities and guide
+collaboration.
+
+## Values and priority level
+
+This is only a subset of the overall priorities, listing only
+those which are at the top of the priority list. Anything on this
+list is very important for the project despite the relative priorities shown.
+
```

Comment:
I propose deleting this for the reasons previously stated: It dilutes rather than strengthens the values statement. And preventing questions along the lines of "what about X as a value?" is not (in my opinion) desirable.

```suggestion
```